packages/math-input/src/index.ts

```diff
@@ -4,16 +4,12 @@
 
 import "../less/main.less";
 
+import Keys from "./data/keys";
+
+export {CursorContext} from "./components/input/cursor-contexts";
 export {default as KeypadInput} from "./components/input/math-input";
 export {keypadElementPropType} from "./components/prop-types";
 export {default as Keypad} from "./components/provided-keypad";
-export {KeypadTypes} from "./consts";
 export {default as KeyConfigs} from "./data/key-configs";
-
-import * as CursorContexts from "./components/input/cursor-contexts";
-
-export {CursorContexts};
-
-export type {KeypadType, KeyType} from "./consts";
-export type {Key} from "./data/keys";
-export type {CursorContext} from "./components/input/cursor-contexts";
+export {Keys};
+export {KeyType, KeypadType} from "./enums";
```

Comment:
This is _definitely_ dropping some exports, but I think it'll be straightforward to adopt these changes in client code (which I'll do when this lands).